tr55/makeMiniAppTable.py

```diff
@@ -55,8 +63,11 @@
             }
         }
 
-        model_out = (simulate_day(cells, precip)
-                     ['unmodified']['distribution'].values()[0])
-        writer.writerow((precip, land_use,
-                         soil_type, model_out['et'],
-                         model_out['inf'], model_out['runoff']))
+        model_out = simulate_day(cells, precip)['unmodified']
```

Comment:
I see we lost the `['distribution'].values()[0]` qualifier. I'm not familiar enough with Python to know if this is simply an optimization, but that's what it must be since it doesn't seem like we've changed the return value of `simulated_day` in this PR.

Could you explain this line please?